pytensor/ifelse.py

```diff
@@ -477,7 +477,8 @@ def cond_make_inplace(fgraph, node):
     Reshape,
     Unbroadcast,
     pt.math.Dot,
-    pt.math.MaxAndArgmax,
+    pt.math.Max,
+    pt.math.Argmax,
```

Comment:
Are these imports being used anywhere? Where was the old `MaxAndArgmax used`?

Reply:
https://github.com/pymc-devs/pytensor/blob/dbe0e09ae2a5f5d0119e0378259f843a95d36abc/pytensor/ifelse.py#L474-L487

The old ```MaxAndArgmax``` is a part of the ```acceptable_ops``` that are used ahead in the ```node_rewritter``` here:

https://github.com/pymc-devs/pytensor/blob/dbe0e09ae2a5f5d0119e0378259f843a95d36abc/pytensor/ifelse.py#L490-L534

Reply:
So we should put these two in the list as well

Reply:
> So we should put these two in the list as well

They already are added in the list

Reply:
My bad I thought this was just the import into the module!